docs/source/release/0.59.0-notes.rst

```diff
@@ -0,0 +1,537 @@
+Add validation capability for user generated towncrier ``.rst`` files.
+======================================================================
+
+Added a validation script for user generated towncrier ``.rst`` files.
+The script will run as a part of towncrier Github workflow automatically
+on every PR.
+
+(`PR-#9335 <https://github.com/numba/numba/pull/9335>`__)
+
+Pull-Requests:
+~~~~~~~~~~~~~~
+
+* PR `#8990 <https://github.com/numba/numba/pull/8990>`_: Removed extra block copying in InlineWorker (`kc611 <https://github.com/kc611>`_)
+* PR `#9048 <https://github.com/numba/numba/pull/9048>`_: Dynamically allocate parfor schedule. (`DrTodd13 <https://github.com/DrTodd13>`_)
+* PR `#9058 <https://github.com/numba/numba/pull/9058>`_: Fix gufunc with multiple outputs (`guilhermeleobas <https://github.com/guilhermeleobas>`_)
+* PR `#9089 <https://github.com/numba/numba/pull/9089>`_: Fix segfault on passing `None` for args in PythonAPI.call (`hellozee <https://github.com/hellozee>`_)
+* PR `#9101 <https://github.com/numba/numba/pull/9101>`_: Add misc script to find missing towncrier news files (`sklam <https://github.com/sklam>`_)
+* PR `#9123 <https://github.com/numba/numba/pull/9123>`_: Implement most ufunc attributes and ufunc.reduce (`guilhermeleobas <https://github.com/guilhermeleobas>`_)
+* PR `#9126 <https://github.com/numba/numba/pull/9126>`_: Add support for np.indices() (`KrisMinchev <https://github.com/KrisMinchev>`_)
+* PR `#9140 <https://github.com/numba/numba/pull/9140>`_: Add support for Polynomial class (`KrisMinchev <https://github.com/KrisMinchev>`_)
+* PR `#9141 <https://github.com/numba/numba/pull/9141>`_: Add support for `as_series()` from `np.polynomial.polyutils` and `polydiv()`, `polyint()`, `polyval()` from `np.polynomial.polynomial` (`KrisMinchev <https://github.com/KrisMinchev>`_)
+* PR `#9142 <https://github.com/numba/numba/pull/9142>`_: Removed out of date comment handled by PR#8338 (`njriasan <https://github.com/njriasan>`_)
+* PR `#9144 <https://github.com/numba/numba/pull/9144>`_: Fix error when literal is wrongly propagated in a PHI node (`guilhermeleobas <https://github.com/guilhermeleobas>`_)
+* PR `#9148 <https://github.com/numba/numba/pull/9148>`_: bump llvmdev dependency to 0.42.0dev for next development cycle (`esc <https://github.com/esc>`_)
+* PR `#9149 <https://github.com/numba/numba/pull/9149>`_: update release checklist post 0.58.0rc1 (`esc <https://github.com/esc>`_)
+* PR `#9152 <https://github.com/numba/numba/pull/9152>`_: Fix old_style error capturing deprecation warnings (`sklam <https://github.com/sklam>`_)
+* PR `#9154 <https://github.com/numba/numba/pull/9154>`_: Add support for np.unwrap() (`KrisMinchev <https://github.com/KrisMinchev>`_)
+* PR `#9155 <https://github.com/numba/numba/pull/9155>`_: Remove unused test.cmd (`sklam <https://github.com/sklam>`_)
+* PR `#9159 <https://github.com/numba/numba/pull/9159>`_: Fix uncaught exception in `find_file()` (`gmarkall <https://github.com/gmarkall>`_)
```

Comment:
```suggestion
```
This was in 0.58.0. Needs removing.
https://github.com/numba/numba/blob/d4460feb8c91213e7b89f97b632d19e34a776cd3/docs/source/release/0.58.0-notes.rst#L520